packages/frontend/src/view/components/Code.tsx

```diff
@@ -1,16 +1,15 @@
 import cx from 'classnames'
 
-export function Code({
-  children,
-  className,
-}: {
+interface Props {
   children: React.ReactNode
   className?: string
-}) {
+}
+
+export function Code({ children, className }: Props) {
   return (
     <pre
       className={cx(
-        'overflow-auto border border-gray-700 bg-gray-900 p-10 font-mono text-gray-500',
+        'overflow-auto border border-gray-700 bg-gray-900 p-10 !text-left font-mono text-gray-500',
```

Comment:
Why is it !important here? Seems to me that it does not affect anything on the page. Correct me if i'm wrong (when debugging even deleting this class doesn't seem to change anything)

Reply:
It is a leftover from the time when I wanted to start at-least some basic rwd ;p But then design pivot appeared